frappe_graphql/frappe_graphql/mutations/set_value.py

```diff
@@ -18,12 +20,16 @@ def resolve_dt(obj, info, *args, **kwargs):
 def set_value_resolver(obj, info: GraphQLResolveInfo, **kwargs):
     doctype = kwargs.get("doctype")
     name = kwargs.get("name")
+    value = kwargs.get('value')
+    fieldname = kwargs.get("fieldname")
+    if frappe.get_meta(doctype).get_field(kwargs.get("fieldname")).fieldtype \
```

Comment:
I think this was missed to use `fieldname` name variable instead of `kwargs.get("fieldname")`

Reply:
refactored 👍